packages/eas-cli/src/project/managedVersions.ts

```diff
@@ -0,0 +1,61 @@
+import { ExpoConfig } from '@expo/config';
```

Comment:
This is a more general suggestion: unify naming. You're using "remote" and "managed" interchangeably. It's less confusing when we use one name for the same thing.